app/src/components/map/MapContainer.tsx

```diff
@@ -37,6 +39,7 @@ export interface IMapContainerProps {
   zoom?: number;
   hideDrawControls?: boolean;
   hideOverlayLayers?: boolean;
+  popupData?: any[];
```

Comment:
Placeholder before I forget: I have some comments, but would be easier to chat about them in discord.  Can sync up tomorrow.

Reply:
sounds good, will discuss and address tomorrow. thanks